test/actions/run_translate.ts

```
@@ -0,0 +1,159 @@
+};
+
+test("test beat translate", async () => {
+  const context = getContext();
+  await translateBeat(1, context, ["fr"]);
+});
+
+
+test("test beat translate", async () => {
+  const context = getContext();
+  context.multiLingual[1].multiLingualTexts = {
+    "fr": {
```

ESLint

[error] Replace `"fr"` with `fr`

(prettier/prettier)